eosiojava/src/main/java/one/block/eosiojava/interfaces/ISerializationProvider.java

```diff
@@ -76,4 +79,24 @@
      * conversion process.
      */
     String serializeAbi(String json) throws SerializeAbiError;
+
+    /**
+     * Convenience method to transform hex string with context free data to an array of data.
+     *
+     * @param hex Hex string representing the context free data to deserialize.
+     * @return Deserialized String representing the data hex.
+     * @throws DeserializeContextFreeDataError A deserialization error is thrown if there are any exceptions during the
+     * conversion process.
+     */
+    List<String> deserializeContextFreeData(String hex) throws DeserializeContextFreeDataError;
```

Comment:
I'm not sure if this is necessary. Would anyone benefit? I didn't do this when implementing the PoC, but I can take a look in to it. Wanted to add it in here to keep it consistent.

Reply:
I might be wrong but I think there could be a case when developer want to modify/update the serialized context free data along with the serialized transaction.

Reply:
We may want to ask Brandon, but I think it makes sense to keep this option.  We have the isModifiable flag which controls whether the transaction can be modified.  The submission may need to be deserialized after modification.

Reply:
Sounds good. Will keep it here then and implement on provider end.